vioscsi/vioscsi.h

```diff
@@ -60,6 +60,7 @@ typedef struct VirtIOBufferDescriptor VIO_SG, *PVIO_SG;
 
 #define REGISTRY_MAX_PH_BREAKS           "PhysicalBreaks"
 #define REGISTRY_ACTION_ON_RESET         "VioscsiActionOnReset"
+#define REGISTRY_RESP_TIME_LIMIT         "ResponseTime"
```

Comment:
Nice! If I'm reading this right, this would be to debug IO that took over X amount of time, which is dictated by the ResponseTime reg key. If that's the only goal, would it make sense to do something like a TraceResponseTime or LogOutlierResponseTime ?

Reply:
Sure, I will change "ResponseTime" to "TraceResponseTime" shortly.
And yes, the main purpose of the patch is to make it easier for the end users and field engineers to 
troubleshoot  vioscsi/backend timing issues in production environments. 
Best,
Vadim.

Reply:
@JonKohler Are you OK with the change?

Reply:
yea, that is OK with me. Thanks for making this happen!